### PR TITLE
insert_file_link will now insert a link created by the prompt if there is no selection present

### PR DESCRIPTION
**Pull request by zdcthomas**
I think that this adds a nice option to the flow of inserting a file link, so that a new file can be created easily if hop is used on the newly generated link.

If there's anything that should be changed at all please let me know! 

Thanks for the awesome plugin!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Simply press a key (`<C-l>` in insert mode by default) and select what you want
 ### Automatic File Link Insertion
 You can use `Telescope neorg insert_file_link` to insert a link to a neorg file.
 Notice that this only works for files in the current workspace.
+Note: the prompt will be used if no selection is chosen.
```
Comment (max397574) [inline README.md:22]: This isn't very clear imo
You should use something like `Note: If no file is selected a link to a file with the name of the prompt value will be inserted. This file will be created if you use the link`

**Comment by max397574** [comment]
Looks good to me.
I didn't even know that files will be created if you have a link to an unexisting files.
Change the `Note` in the readme a little bit (it doesn't have to be 1:1 what I suggested but should include the same info) and then I can merge.

**Comment by max397574** [comment]
Thank you for the pr

**Comment by zdcthomas** [comment]
Good call on the better wording! I also added a link to hop for clarity.

**Pull request merged**